Testing/VelaptorTests/NativeInterop/OpenGL/OpenGLServiceTests.cs

```diff
@@ -362,7 +381,7 @@ public void LabelShaderProgram_WhenInvoked_LabelsShaderProgram()
         service.LabelShaderProgram(123, label);
 
         // Assert
-        this.mockGLInvoker.Verify(m => m.ObjectLabel(GLObjectIdentifier.Program, 123, (uint)label.Length, label));
+        this.mockGLInvoker.Received().ObjectLabel(GLObjectIdentifier.Program, 123, (uint)label.Length, label);
```

Comment:
Please refactor the `Received()` method to take the parameter value of `1`.

**Explanation:**
The reason for this is that if you do not use a number as the parameter, then it will check that the method was invoked _**AT LEAST 1 OR MORE TIMES**_ instead of exactly once.  We need to make sure that it is only called _**EXACTLY 1 TIME**_.

90% of the time, we need to be explicit about how many times methods are tested for invokes.

```suggestion
        this.mockGLInvoker.Received(1).ObjectLabel(GLObjectIdentifier.Program, 123, (uint)label.Length, label);
```